src/home/domain.ts

```diff
@@ -211,7 +211,7 @@ const updatePage = (selector: string) => {
 };
 
 const updateProgress = (value: number) => {
-  utils.queryAll("#progress .progress").forEach((x) => { void ui(x, value); });
+  utils.queryAll("#progress progress[value]").forEach((x: any) => { x.value = value; });
```

Comment:
Maybe we should avoid using any ?

Not sure if its the best option but we could try:

const updateProgress = (value: number) => {
  utils.queryAll("#progress progress[value]").forEach((x) => {
    const progressElement = x as HTMLProgressElement;
    progressElement.value = value;
  });
};